superset-frontend/src/dashboard/components/FiltersBadge/index.tsx

```diff
@@ -84,16 +84,15 @@ const StyledFilterCount = styled.div`
 
 const StyledBadge = styled(Badge)`
   ${({ theme }) => `
-    vertical-align: middle;
     margin-left: ${theme.gridUnit * 2}px;
-    &>sup {
-      padding: 0 ${theme.gridUnit}px;
+    &>sup.ant-badge-count {
```

Comment:
This is another example of a forced customization to meet Superset’s requirements to fit the Badge within the available space. Such customizations are undesirable as they introduce maintenance costs. Another issue to consider is the reliance on a specific className, which poses risks if this className is removed or changed in future versions of Ant Design.